### PR TITLE
Bulk organisation updater improvements

### PR DESCRIPTION
**Pull request by cbaines**

### CODE REVIEW

File: lib/data_hygiene/bulk_organisation_updater.rb [hunk 1]
```diff
@@ -35,10 +35,14 @@ def process_row(row)
 
     def find_document(row)
       slug = row.fetch("Slug")
+
+      return if slug.nil?
+      return if slug.strip.empty?
```
Comment (AlanGabbianelli) [inline lib/data_hygiene/bulk_organisation_updater.rb:40]: Since `"  ".blank?` is `true`, we can do
```suggestion
      return if slug.blank?
```

Comment (cbaines) [reply lib/data_hygiene/bulk_organisation_updater.rb:40]: Sure, do you want to make the change? I don't know how to use the "suggest change" thing on GitHub.

Comment (AlanGabbianelli) [reply lib/data_hygiene/bulk_organisation_updater.rb:40]: If I make the change then I can't approve the PR. If the change looks good to you, you can just edit the commit and force push.

Comment (cbaines) [reply lib/data_hygiene/bulk_organisation_updater.rb]: Ok, I've incorporated this in to the first commit now.

**Review by AlanGabbianelli — APPROVED** [review]

**Pull request merged**